net.sourceforge.projects.vht.yml

```diff
@@ -0,0 +1,19 @@
+id: net.sourceforge.projects.vht
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: vht
+finish-args:
+  - --filesystem=home
+  - --filesystem=xdg-run/pipewire-0
+  - --socket=wayland
+  - --socket=fallback-x11
+modules:
+  - name: vht
+    buildsystem: simple
+    build-commands:
+      - pip3 install . --no-build-isolation --prefix=${FLATPAK_DEST}
```

Comment:
build is offline so it won't find the dependencies.

Reply:
Sorry, but I'm not sure what that means (I'm new to this). Is this about --no-build-isolation?

Reply:
do you try to build it locally?

Reply:
it builds and runs fine locally